src/moduleResolver/ModuleResolver.ts

```diff
@@ -80,22 +80,32 @@ function initModule(props: { absPath: string; bundleContext: IBundleContext; ctx
     if (module.isExecutable) {
       module.parse();
       const transformerResult = module.transpile();
+
       for (const item of transformerResult.requireStatementCollection) {
-        const source = item.statement.arguments[0].value;
-
-        const resolvedModule = resolve({
-          bundleContext: props.bundleContext,
-          ctx: props.ctx,
-          importType: item.importType,
-          parent: module,
-          statement: source,
-        });
-
-        module.moduleSourceRefs[source] = resolvedModule;
-        // re-writing the reference
-        item.statement.callee.name = BUNDLE_RUNTIME_NAMES.ARG_REQUIRE_FUNCTION;
-        item.statement.arguments[0].value = resolvedModule.id;
-        delete item.statement.arguments[0].raw;
+        // no value.. import() or require();
+        if (!item.statement.arguments[0]) continue;
```

Comment:
we still need to make an `ARG_REQUIRE_FUNCTION` of it. And pass `-1` or `0` as an argument. (IMHO) so we could pick it up and runtime and throw an error